File: 4-Patron/php/Skin.php
<?php
class Skin
{
  private $nombreSkin;
  private $tipoSkin;

  function __construct($nSkin, $tSkin)
  {
    $this->nombreSkin = $nskin;
    $this->tipoSkin = $tskin;
  }

  function getNombreSkin()
  {
    return $this->nombreSkin;
  }

  function setNombreSkin($nNombre)
  {
    $this->nombreSkin = $nNombre;
  }

  function getTipoSkin()
  {
    return $this->tipoSkin;
  }

  function setTipoSkin($nTipo)
  {
    $this->tipoSkin = $nTipo;
  }

}
?>

File: 4-Patron/Java/Skin.java
package patron4;

public class Skin {
    private String nombreSkin;
    private String tipoSkin;

    public Skin(String nombreSkin, String tipoSkin) {
        this.nombreSkin = nombreSkin;
        this.tipoSkin = tipoSkin;
    }

    public String getNombreSkin() {
        return nombreSkin;
    }

    public String getTipoSkin() {
        return tipoSkin;
    }

    public void setNombreSkin(String nombreSkin) {
        this.nombreSkin = nombreSkin;
    }

    public void setTipoSkin(String tipoSkin) {
        this.tipoSkin = tipoSkin;
    }
    
    
}
